storage/gcsemu/remote_test.go

```diff
@@ -33,7 +33,17 @@ func TestRealStore(t *testing.T) {
 	assert.NilError(t, err)
 
 	t.Parallel()
-	for _, tc := range testCases {
+
+	// these tests are not applicable to real storage
```

Comment:
I left it commented here because its not simple to run this test on real gcs, mainly because the second step of the test is to create a bucket